docs/src/atomistic/outputs.rst

```diff
@@ -119,3 +119,45 @@ The following gradients can be defined and requested with
     - ``["xyz_1", "xyz_2"]``
     - Both ``"xyz_1"`` and ``"xyz_2"`` have values ``[0, 1, 2]``, and correspond
       to the two axes of the 3x3 strain matrix :math:`\epsilon`.
+
+
+Energy ensemble
+^^^^^^^^^^^^^^^
+
+An ensemble of energies is associated with the ``"energy_ensemble"`` key in the
+model outputs, and must have the following metadata:
```

Comment:
I'm split whether we should duplicate the information that is identical to `energy` here, or say "keys: same as `energy`", "samples: same as `energy`", ... and only specify the bit that changes.

Do you have a preference?

Reply:
I would definitely prefer "same as energies"

Reply:
let's do this then

Reply:
Should be done